Cargo.toml

```diff
@@ -3,10 +3,19 @@ name = "geotiff"
 description = "A GeoTIFF library for Rust"
 version = "0.0.2"
 edition = "2021"
-authors = ["Dominik Bucher <dobucher@ethz.ch>"]
+authors = [
+    "Dominik Bucher <dobucher@ethz.ch>",
+    "Gunnar Schulze <gunnar.schulze@gmail.com>"
+]
 repository = "https://github.com/georust/geotiff"
 
 [dependencies]
+delaunator = "1.0"
+geo-index = "0.1"
```

Comment:
Thoughts on putting these behind a feature flag, considering that they're only used for GeoTIFF with GCPs (that are relatively rare)?

Reply:
Yes, makes sense. What should it be called, "gcp"?

Reply:
I'm not very good with names :sweat_smile: Maybe "tiepoints"?